src/mgmt/srv.c

```diff
@@ -410,15 +411,64 @@ process_join_request(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 }
 
 static int
-create_ownership_props(daos_prop_t **out_prop, char *owner, char *owner_grp)
+ace_strs_to_acl(char **ace_strs, size_t ace_nr, struct daos_acl **acl)
 {
-	char	       *out_owner = NULL;
-	char	       *out_owner_grp = NULL;
-	daos_prop_t    *new_prop = NULL;
+	struct daos_ace	**tmp_aces;
+	struct daos_acl	*tmp_acl = NULL;
```

Comment:
I don't think this needs to be initialised now.

Reply:
Fixed